nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EvaluateXPath.java

```diff
@@ -129,8 +129,10 @@ public class EvaluateXPath extends AbstractProcessor {
             .build();
 
     public static final PropertyDescriptor VALIDATE_DTD = new PropertyDescriptor.Builder()
+            .displayName("Allow Document Type Declaration")
```

Comment:
Recommend naming this `Allow DTD` to keep it similar to the previous name.
```suggestion
            .displayName("Allow DTD")
```